python/python/bystro/vcf_utils/sim_data_generator.py

```diff
@@ -0,0 +1,61 @@
+"""VCF index and data simulator for testing analysis modules."""
+
+import random
+from io import StringIO
+
+import pandas as pd
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
+
+
+def generate_random_vcf_index():
+    """Generate autosomal chr vcf variant IDs to use for testing."""
+    random_chr = random.randint(1, 22 + 1)
+    random_pos = random.randint(1, 1000000)
+    random_ref = random.choice(["A", "T", "C", "G"])
+    random_alt = random.choice([letter for letter in ["A", "T", "C", "G"] if letter != random_ref])
+    return f"{random_chr}:{random_pos}:{random_ref}:{random_alt}"
+
+
+def generate_simulated_vcf(
+    num_samples:int, num_vars:int
+) -> tuple[pd.DataFrame, str]:
```

Comment:
I would push back against this return type, actually.  If we're simulating VCFs, seems like we'd get more mileage out of a textual type (probably just `str`) rather than a df so we can test df parsing.